specification/paths/Carriers-carrier_id-PickupDropoffLocations-latitude-longitude.json

```diff
@@ -0,0 +1,121 @@
+{
+  "parameters": [
+    {
+      "$ref": "#/components/parameters/path-carrier_id"
+    },
+    {
+      "$ref": "#/components/parameters/path-latitude"
+    },
+    {
+      "$ref": "#/components/parameters/path-longitude"
+    }
+  ],
+  "get": {
+    "tags": [
+      "Carriers"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+        ]
+      }
+    ],
+    "summary": "Get office locations from a carrier by geolocation.",
+    "description": "This endpoint retrieves pick-up and drop-off locations by geolocation offered by the supplied carrier.<ul><li>Locations of the type `pick-up` can be used with services offering this as their `delivery_method`.</li><li>Locations of the type `drop-off` can be used with services offering this as their `handover_method`.</li></ul>",
+    "parameters": [
+      {
+        "name": "filter[contract]",
+        "in": "query",
+        "description": "Specific contract (id) to use for authentication with the carrier. If none is specified then the first active contract will be used.",
+        "schema": {
+          "type": "string"
+        }
+      },
+      {
+        "name": "filter[radius]",
+        "in": "query",
+        "description": "Specific contract (id) to use for authentication with the carrier.",
```

Comment:
I think you forgot to change this description :eyes:

Reply:
Damn, I am loosing focus!